wildstore-cli/src/main/java/edu/sjsu/wildstore/Main.java

```diff
@@ -80,6 +80,16 @@ public void share(
                 co.err().printf("%s: %s\n", message, Arrays.toString(fileNames));
             } catch (WebClientResponseException e) {
                 throw new CommandLine.ExecutionException(co.cmd(), e.getMessage(), null);
+            } catch (NullPointerException e) {
+                if (fileNames == null) {
+                    throw new CommandLine.ExecutionException(co.cmd(), "fileNames is missing or not provided");
```

Comment:
i think if you add a arity="1..*" parameter to line 59, you will not need this check